README.md

```diff
@@ -41,66 +45,79 @@ git clone https://github.com/o3de/o3de.git
 
 #### Optional
 
-*   Wwise version 2021.1.1.7601 minimum: [https://www.audiokinetic.com/download/](https://www.audiokinetic.com/download/)
-    *   Note: This requires registration and installation of a client application to download
-    *   Note: It is generally okay to use a more recent version of Wwise, but some SDK updates will require code changes
-    *   Make sure to select the `SDK(C++)` component during installation of Wwise
-    *   CMake can find the Wwise install location in two ways:
-        *   The `LY_WWISE_INSTALL_PATH` CMake cache variable -- this is checked first
-        *   The `WWISEROOT` environment variable which is set when installing Wwise SDK
+*   Wwise audio SDK
+    *   For the latest version requirements and setup instructions, refer to the [Wwise Audio Engine Gem](https://o3de.org/docs/user-guide/gems/reference/audio/wwise/audio-engine-wwise/) reference in the documentation.
 
-### Quick Start Build Steps
+### Quick start engine setup
 
-1.  Create a writable folder to cache 3rd Party dependencies. You can also use this to store other redistributable SDKs.
+To set up a project-centric source engine, complete the following steps. For other build options, refer to [Setting up O3DE from GitHub](https://o3de.org/docs/welcome-guide/setup/setup-from-github/) in the documentation.
+
+1.  Create a writable folder to cache downloadable packages. You can also use this to store other redistributable SDKs.
```

Comment:
might be good to keep something in hear about 3rdparty packages, e.g. "Create a writable folder to cache downloadable 3rd Party packages"